### PR TITLE
feat:Theme icon replacement

### PR DESCRIPTION
**Pull request by ianxinnew**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **Style**
  - Adjusted padding, border-radius, and font-size values for better UI consistency in the design plugin section.
  - Updated icon dimensions and switched from `.png` to `.svg` for improved graphics rendering in the robot plugin.
  - Standardized icon sizes and border-radius in the media toolbar for a more cohesive look and feel.

- **Bug Fixes**
  - Replaced dynamic language icon with a static string to ensure consistent icon display in the i18n plugin.

### CODE REVIEW

File: packages/design-core/assets/add-folder.svg [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-            <line x1="9.2658175" y1="9.47197958" x2="11.9908175" y2="9.47197958" id="直线-2"></line>
-            <line x1="9.2658175" y1="9.47197958" x2="11.9908175" y2="9.47197958" id="直线-2备份" transform="translate(10.628317, 9.471980) rotate(90.000000) translate(-10.628317, -9.471980) "></line>
-            <line x1="0.00040875" y1="4.21663313" x2="11.9904087" y2="4.21663313" id="直线"></line>
-        </g>
-    </g>
-</svg>
\ No newline at end of file
+<svg width="16.267334" height="16.267334" viewBox="0 0 16.2673 16.2673" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+	<desc>
+			Created with Pixso.
```
Comment (wenmine) [inline packages/design-core/assets/add-folder.svg]: 这种没用的注释去掉，把整个\<desc>去掉

File: packages/design-core/assets/add-folder.svg [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+	</desc>
+	<defs/>
+	<mask id="mask413_10171" mask-type="alpha" maskUnits="userSpaceOnUse" x="0.000000" y="0.000000" width="16.267334" height="16.267334">
+		<rect id="新建文件夹-folder-add" x="0.269531" width="16.000000" height="16.000000" transform="rotate(0.965286 0.269531 0.000000)" fill="#000000" fill-opacity="1.000000"/>
```
Comment (wenmine) [inline packages/design-core/assets/add-folder.svg]: fill="#000000"，#000000 不应该写死，应该用currentColor

File: packages/design-core/assets/datepick.svg [hunk 3]
```diff
@@ -1,17 +1,8 @@
...
-    c0-0.8-0.4-1.5-1.1-1.8c-0.8-0.4-1.7-0.6-2.6-0.6c-2.4,0-3.7,0.8-3.7,2.4v2.3h2.9v-2.3c0-0.3,0.2-0.4,0.6-0.4s0.6,0.1,0.6,0.4v2.4
-    c0,0.5-0.2,0.8-1.2,0.8v1.5c0.5,0,0.7,0.1,0.9,0.2c0.2,0.1,0.3,0.4,0.3,0.7v3.1c0,0.3-0.2,0.5-0.6,0.5s-0.6-0.2-0.6-0.5v-3h-3v3
-    C24.7,45.4,25.9,46.2,28.4,46.2z"/>
-    <path class="st0" d="M57,11.3h-9.7V8h-4v3.4H34V8h-4v3.4h-9.3V8h-4v3.4H7c-1.1,0-2,0.9-2,2v40.3c0,1.1,0.9,2,2,2h50
-    c1.1,0,2-0.9,2-2V13.3C59,12.2,58.1,11.3,57,11.3z M55,51.6H9V27.8h46V51.6z M55,24.8H9v-9.5h7.7v4.2h4v-4.2H30v4.2h4v-4.2h9.3v4.2
-    h4v-4.2H55V24.8z"/>
-  </g>
+<svg width="20.000000" height="20.661133" viewBox="0 0 20 20.6611" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+	<desc>
+			Created with Pixso.
```
Comment (wenmine) [inline packages/design-core/assets/datepick.svg:3]: 上述两点，需要对每个svg文件都审视一下

File: packages/layout/src/DesignPlugins.vue [hunk 4]
```diff
@@ -284,21 +284,21 @@ export default {
...
-        font-size: 22px;
+        font-size: 18px;
       }
     }
   }
 }
 
 :deep(.panel-svg) {
-  font-size: 22px;
+  font-size: 18px;
```
Comment (wenmine) [inline packages/layout/src/DesignPlugins.vue:298]: 字体大小要用变量

**Review by hexqi — APPROVED** [review]

**Pull request merged**